docs/GettingStartedDocs/Linux_vscode.md

```diff
@@ -0,0 +1,130 @@
+# Building And Debugging Using Visual Studio Code for Linux Development
+
+This document provides a brief overview of how to build and debug Open Enclave applications using VS Code on Linux.
+
+## Where to Run VS Code?
+
+There are two ways of running VS Code for development on Linux.
+1. Run VS Code directly on your Linux system, which will give you a native Linux display of VS Code.
+2. Run VS Code from any modern Windows system and use the `Remote SSH` extension in VS Code to connect to a Linux system you wish to develop on.
+  a. For more information on how to use the `Remote SSH` extension, please follow this blog post here: [https://code.visualstudio.com/blogs/2019/07/25/remote-ssh](https://code.visualstudio.com/blogs/2019/07/25/remote-ssh)
+
+## Install VS Code
+
+The latest version of Visual Studio Code can be installed from [https://code.visualstudio.com/](https://code.visualstudio.com/)
+
+## Install VS Code Extensions
+
+Install the following VS Code extensions. If you are using the Remote SSH, ensure they are installed remotely too by clicking  Click on an image to navigate to the Visual Studio Code Marketplace page for the extension.
```

Comment:
Some words seem to have been cut off "ensure they are installed remotely too by clicking Click..."

Reply:
Thanks Anand! I wonder how that happened here. I don't remember writing this. Maybe I accidentally pasted something...